### PR TITLE
Add method on version repository to determine a series of Java versions

### LINKED ISSUE #8: Add support for getting a Java version series
In order to hide older versions of a version series (e.g. all Java `11`, Java `8`, Java `9` versions) a new method should be provided which, when given

* a `platform`
* a `majorVersion` (e.g. `11`)
* a `vendorSuffix` (e.g. `zulu-fx`, `zulu`)

returns a list of `existing` versions, which can then be `hidden`.

### PR DESCRIPTION
**Pull request by helpermethod**
Fixes #8.

### CODE REVIEW

File: src/test/scala/io/sdkman/repos/VersionsRepoSpec.scala [hunk 1]
```diff
@@ -199,6 +200,51 @@ class VersionsRepoSpec extends AnyWordSpec with Matchers with BeforeAndAfter wit
         }
       }
     }
+
+    "attempt to find all versions by platform where version starts with a major version and ends on a vendor suffix" in new TestRepo with TableDrivenPropertyChecks {
```
Comment (marc0der) [inline src/test/scala/io/sdkman/repos/VersionsRepoSpec.scala]: How about we mix in the table driven property checks trait at the specification level. It feels like it should belong at the test level. wdyt?

**Comment by marc0der** [comment]
Looks great,  just one minor thing in the test.

**Comment by helpermethod** [comment]
Done!

**Comment by marc0der** [comment]
@helpermethod This has now been released as `v1.6` of the library.

**Pull request merged**